packages/gestalt/src/utils/wrapWithComponent.js

```diff
@@ -28,10 +28,8 @@ function isElementOfType<P>(element: ?Node, Component: ChildrenArray<ComponentTy
   const { type } = element;
   const Components = Array.isArray(Component) ? Component : [Component];
 
-  return Components.some(
-    // $FlowFixMe[incompatible-call]
-    (AComponent) => typeof type !== 'string' && isComponent(AComponent, type),
-  );
+  // $FlowExpectedError[incompatible-call]
```

Comment:
Flow is still unhappy about how we're handling `type` here, but that's expected: Flow wants us to know more than we possibly could